client/src/containers/BBLPage.tsx

```diff
@@ -87,7 +88,7 @@ export default class BBLPage extends Component<BBLPageProps, State> {
     }
 
     APIClient.getBuildingInfo(fullBBL)
-      .then((results) => {
+      .then((results: BuildingInfoResults) => {
```

Comment:
Yeah I was confused about it too! I originally didn't have these but then got this error in CircleCI: 
```
src/containers/HomePage.tsx:103:16 - error TS7006: Parameter 'results' implicitly has an 'any' type.

103         .then((results) => {
 ```

I'm wondering though whether that error actually had to do with a previous error about the mistyping of Borough in certain functions, and was just not specifying things clearly. I'll play around and see if we can avoid adding these redundant typings...

Reply:
Huh, I just removed the typings and we are no longer getting the error. It must have been one of those weird bugs where on type error cascades in to many more?

Reply:
oh fascinating!  huh yeah i dunno but that's great that you were able to remove this stuff!  it especially worried me not so much because of the non-DRYness but because if we accidentally put the wrong explicit typing in, the compiler wouldn't have any way of telling us we were wrong.